includes/class-wc-admin-install.php

```diff
@@ -265,6 +271,7 @@ public static function create_events() {
 		if ( ! wp_next_scheduled( 'wc_admin_daily' ) ) {
 			wp_schedule_event( time(), 'daily', 'wc_admin_daily' );
 		}
+		wp_schedule_single_event( time() + 10, 'generate_category_lookup_table' );
```

Comment:
I think this should probably also be run during import.  We also may want to remove any scheduled events in the `on_deactivation()` hook.